Week 4/Enigma-java/src/main/java/Rotor.java

```diff
@@ -0,0 +1,46 @@
+import java.util.List;
+
+public class Rotor 
+{
+
+    public String rotorEncrypt(String msg, List<String> rotors)
+    {
+        final String alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+        String encryptedMsg = "";
+        for (int i = 0; i < rotors.size(); i++)
+        {
+            //Swaps/resets variables after the first loop
+            if(i > 0)
+            {
+                msg = encryptedMsg;
+                encryptedMsg = "";
+            }
+
+            for(char letter: msg.toCharArray())
```

Comment:
## More Comments
Although the file does contain some comments, more descriptive and informative comments would be make it easier for other members to understand the logic behind the code and how each function operates